src/main/java/uk/gov/digital/ho/hocs/workflow/api/WorkflowService.java

```diff
@@ -69,53 +69,48 @@ public WorkflowService(CaseworkClient caseworkClient,
 
     public CreateCaseResponse createCase(String caseDataType, LocalDate dateReceived, List<DocumentSummary> documents, UUID userUUID, UUID fromCaseUUID, Map<String, String> receivedData) {
         // Create a case in the casework service in order to get a reference back to display to the user.
-        Map<String, String> data = new HashMap<>(receivedData);
         CreateCaseworkCorrespondentRequest correspondentRequest = null;
```

Comment:
Defaults to `null` so not needed.

Reply:
If the assign to null is removed the null check on line 115 warns that the variable might not have been initialised.